README.md

```diff
@@ -2,53 +2,25 @@
 
 # Hello Microverse
 
-> This project of hello microverse project about github flow.
-
-![screenshot](./app_screenshot.png)
+> This is a basic project called Hello Microverse about basic github flow.
 
 
 
 ## Built With
```

Comment:
- [ ] **OPTIONAL:** I kindly advise you to name feature branches with descriptive names( do not include your GitHub handle), for example, `added_comments_section`, `set_up_github_flow`, it will make your projects understandable in the future.